packages/block-library/src/query-pagination-previous/edit.js

```diff
@@ -34,8 +34,8 @@ export default function QueryPaginationPreviousEdit( {
 				__experimentalVersion={ 2 }
 				tagName="span"
 				aria-label={ __( 'Previous page link' ) }
-				placeholder={ __( 'Previous Page' ) }
-				value={ label }
+				placeholder={ showLabel ? __( 'Previous Page' ) : '' }
```

Comment:
Why still render the `PlainText` and only update the placeholder. This way you can still set it's value.

Reply:
This makes sense, thank you. This should be addressed here: https://github.com/WordPress/gutenberg/pull/51165/commits/2aaf07dae74c85979fb0396c23cea2e2af23317c